app/services/team.ts

```diff
@@ -16,7 +25,7 @@ export const createTeam = async (data: any): Promise<[Team, null] | [null, Axios
   try {
     const result = await instance.post('teams', data).then((res) => res.data);
     return [result, null];
-  } catch (err) {
+  } catch (err: any) {
```

Comment:
Hmm didn't realize we needed to declare type on errors in typescript.  Good to know.

Reply:
right, Typescripts complaints of no type definition for the error object from `try... catch` blocks.